app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/DrawMicrophoneButton.kt

```diff
@@ -0,0 +1,36 @@
+package com.wire.android.ui.calling.controlButtons
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.wire.android.R
+import com.wire.android.ui.theme.wireDimensions
+
+@Composable
+fun DrawMicrophoneButton() {
+    var isCallMuted by remember { mutableStateOf(true) }
```

Comment:
inside the state object you got the flag if the conv. is either muted or not, therefore I think it makes more sense to give the possibility to change the status of the button to the viewmodel, by adding a lambda and boolean value into the @Composable function

Reply:
yeah to be done in next PR 😄